docs/tutorials/denom-allolist-proposal.md

```diff
@@ -0,0 +1,40 @@
+# Currency Allow list Proposals
```

Comment:
"Denom Allowlist Proposal" or "Currency Allowlist Proposal"? The title should be consistent.

```suggestion
# Currency Allowlist Proposal
```